### ISSUE TITLE
A11y simulation mode

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#781]
A11y module starts to put in motion the block settings needed to help people self identify as needing things like better fonts for dyslexia. We always hear that those most passionate about A11y issues are those directly affected by it. The empathy generated by seeing someone struggle with poorly made / inaccessible technology creates champions for the cause who can do something about it.

Let's take that empathy issue and democratize it with, built into elmsln core, disability simulator. For example; you might not know why someone would want the inverted colors / enhanced contrast / text size and what not; but you might know how to select "Dyslexic" from a dropdown and have it select the optimal configuration for someone with that condition. This can also help you do things like author content that has video and images and then come to realize that when you put it in "color blind" mode that your images don't work.

Much of the time it's a lack of awareness as to why people cause issues for those with A11y needs; let's fix that through simulation.

**Comment by btopro** [comment]
http://geon.github.io/programming/2016/03/03/dsxyliea

**Comment by btopro** [comment]
http://u.osu.edu/universaldesign/personas/
